new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java

```diff
@@ -507,6 +741,72 @@ public void testAddMiiConfiguredCluster() {
     logger.info("Found new managed server configuration");
   }
 
+  /**
+   * Start a WebLogic domain with model-in-imge.
+   * Patch domain CRD with new webLogicCredentialsSecret.
```

Comment:
Patch the domain CRD with new credentials secret.

Reply:
modified:  Patch the domain CRD with a new credentials secret